services/web/server/tests/unit/with_dbs/test_projects.py

```diff
@@ -728,3 +730,48 @@ def _extract(dikt, keys):
     else:
         mock_director_api_stop_services.assert_not_called()
         mock_storage_api_delete_data_folders_of_project_node.assert_not_called()
+
+
+@pytest.mark.parametrize("user_role,expected", [
+    (UserRole.USER, web.HTTPOk)
+])
+async def test_tags_to_studies(client, logged_user, user_project, expected, test_tags_data):
+    # Add test tags
+    tags = test_tags_data
+    added_tags = []
+    for tag in tags:
+        url = client.app.router["create_tag"].url_for()
+        resp = await client.post(url, json=tag)
+        added_tag, _ = await assert_status(resp, expected)
+        added_tags.append(added_tag)
+        # Add tag to study
+        url = client.app.router["add_tag"].url_for(study_uuid=user_project.get("uuid"), tag_id=str(added_tag.get("id")))
+        resp = await client.put(url)
+        data, _ = await assert_status(resp, expected)
+        # Tag is included in response
+        assert added_tag.get("id") in data.get("tags")
+
+    # Delete tag0
+    url = client.app.router["delete_tag"].url_for(tag_id=str(added_tags[0].get("id")))
```

Comment:
delete shall return 204

Reply:
it does! only removing a tag from a project returns the updated project (in that case we are only removing a relationship)... we use that response for caching the new modified project in the frontend

Reply:
Aha! I got confused because you parametrize the test with an ``expected=200``!